pkgs/applications/graphics/unigine_heaven/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  libX11,
+  libXext,
+  libXrandr,
+  freetype,
+  fontconfig,
+  libXrender,
+  libXinerama,
+  autoPatchelfHook,
+  libglvnd,
+  openal,
+}:
+
+let
+  version = "4.0";
+in
+
+stdenv.mkDerivation {
+  pname = "Unigine_Heaven";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://assets.unigine.com/d/Unigine_Heaven-${version}.run";
+    sha256 = "19rndwwxnb9k2nw9h004hyrmr419471s0fp25yzvvc6rkd521c0v";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  buildInputs = [
+    libX11
+    stdenv.cc.cc
```

Comment:
Is `stdenv.cc.cc` really needed here? The compiler and its related tools should be available by default since `stdenv.mkDerivation` is used.

Reply:
Yes. When I remove it then package doesn't build. It writes 
``
autoPatchelfHook could not satisfy dependency libstdc++.so.6 wanted by /nix/store/2n7ggq69xfbcbg5lgzcfb1kchiz9wjgd-Unigine_Heaven-4.0/bin/libQtGuiUnigine_x64.so.4
``